### PR TITLE
Discovery api umbrella migration

### PR DESCRIPTION
**Pull request by kcarra**
Prestige upgrade to 1.x

### CODE REVIEW

File: apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs [hunk 1]
```diff
@@ -5,34 +5,45 @@ defmodule Pipeline.Writer.TableWriterTest do
...
 
       schema = [
         %{name: "one", type: "list", itemType: "string"},
         %{name: "two", type: "map", subSchema: [%{name: "three", type: "decimal(18,3)"}]},
-        %{name: "four", type: "list", itemType: "map", subSchema: [%{name: "five", type: "integer"}]}
+        %{name: "four", type: "list", itemType: "map", subSchema: [%{name: "five", type: "decimal(18,3)"}]}
       ]
 
-      dataset = TDG.create_dataset(%{technical: %{systemName: "org_name__dataset_name", schema: schema}})
+      dataset = TDG.create_dataset(%{technical: %{systemName: "org_name_dataset_name", schema: schema}})
```
Comment (rajones-hntb) [inline apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs]: Was removing this underscore intentional?

File: apps/reaper/test/unit/reaper/event/handlers/dataset_update_test.exs [hunk 2]
```diff
@@ -99,18 +99,23 @@ defmodule Reaper.Event.Handlers.DatasetUpdateTest do
     end
 
     data_test "jobs that are added to quantum when cadence is a cron expression work, even with missing optional fields" do
-      {:ok, cadence} = Crontab.CronExpression.Parser.parse("*/5 * * * * * *", true) # gotta go fast
+      # gotta go fast
```
Comment (dunson062786) [inline apps/reaper/test/unit/reaper/event/handlers/dataset_update_test.exs]: please remove this comment

File: apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs (hunk 1 above)
Comment (mcsearchin) [reply apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs]: i believe this is the naming convention so i'd be surprised if this isn't causing a test to fail ~Jeff

File: apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs [hunk 3]
```diff
@@ -142,67 +157,79 @@ defmodule Pipeline.Writer.TableWriterTest do
...
       end)
     end
   end
 
   describe "compact/1" do
-    test "compacts a table without changing data" do
+    test "compacts a table without changing data", %{session: session} do
       sub = [%{name: "three", type: "boolean"}]
-      schema = [%{name: "one", type: "list", itemType: "integer"}, %{name: "two", type: "map", subSchema: sub}]
+      schema = [%{name: "one", type: "list", itemType: "decimal"}, %{name: "two", type: "map", subSchema: sub}]
```
Comment (mcsearchin) [inline apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs]: why are we changing these types? this type has proven to be ... problematic in the past. ~Jeff

File: apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs [hunk 4]
```diff
@@ -142,67 +157,79 @@ defmodule Pipeline.Writer.TableWriterTest do
...
+    test "fails without altering state if it was going to change data", %{session: session} do
       allow Compaction.measure(any(), any()), return: {6, 10}, meck_options: [:passthrough]
 
       schema = [%{name: "abc", type: "string"}]
       dataset = TDG.create_dataset(%{technical: %{schema: schema, systemName: "xyz"}})
 
       TableWriter.init(table: dataset.technical.systemName, schema: schema)
 
-      Enum.each(1..10, fn n ->
+      Enum.each(1..15, fn n ->
```
Comment (mcsearchin) [inline apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs]: any particular reason we increased the number of rows here and above? ~Jeff

File: apps/reaper/test/unit/reaper/event/handlers/dataset_update_test.exs (hunk 2 above)
Comment (kennyatpillar) [reply apps/reaper/test/unit/reaper/event/handlers/dataset_update_test.exs]: Should work this into a test somehow.

File: apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs (hunk 4 above)
Comment (kcarra) [reply apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs]: It seems to be an issue with prestige, opened an [issue](https://github.com/smartcitiesdata/prestige/issues/19) for it.

**Comment by kcarra** [comment]
Closing this pr as @NathanielScottStevens and I will be opening another for the entire discovery api migration

**Pull request closed without merging**